app/routes/analytics.js

```diff
@@ -12,9 +12,9 @@ router.post('/fingerprint', async (req, res) => {
         const hashedFingerprint = hashFingerprint(fingerprint)
         const route = await Counter.findOne({url: currentUrl})
         if(!route) {
-            const articleId = currentUrl.substr(10)
-            if(!ObjectId.isValid(articleId)) return res.sendStatus(500)      
-            await Counter.create({ url: currentUrl, viewCount: 1, visitedUsers: [hashedFingerprint], articleId})
+            const articleLink = currentUrl.substr(10)
```

Comment:
What is happening here

Reply:
I am subtracting "/articles/" from the pathname of /articles/:importantpart

Reply:
That's very tricky, you wanted to do i18n later, imagine what would happen if you'd do if you'd start translating links. Can't you at least do `replace('/article', '')` or sth else lesss enigmatic than `substr(10)`?

Reply:
Sure thing.